message_tools/find_invalid_address_case_id_by_uprn.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import csv
+from pathlib import Path
+
+import requests
+from google.cloud import storage
+from requests import HTTPError
+
+from config import Config
+
+
+def read_invalid_address_uprn_file(file_to_process, project_id):
+    case_id_list = []
+    with open(file_to_process, encoding="utf-8") as open_file_to_process:
+        file_reader = csv.DictReader(open_file_to_process, delimiter=',')
+        print(file_reader)
+
+        for line_number, row in enumerate(file_reader, 1):
+            case_id_list.extend(get_case_id_from_case_api(row['UPRN']))
+
+    address_delta_file = write_invalid_addresses_case_id_file(case_id_list, Path(file_to_process))
```

Comment:
To avoid confusion I think these names need to be flipped, the address_delta_file is what is being fed _into_ this, it's generating an invalid addresses file.